### ISSUE TITLE
Is there a way please to clear IP bans without restarting the room?

### ISSUE THREAD

**Issue by SpiderSuave** [m1k1o/neko#395]
Until there's a Neko-Rooms or admin option to clear bans without restarting the room, is there something I can do in the meantime on the server side to clear bans while the room is still running?

Thank you for any help.

**Comment by m1k1o** [comment]
No, currently there is no way. You could, however, restart just the neko binary inside room. So browser, X server and everything else stays running and all users should just reconnect.

**Comment by SpiderSuave** [comment]
> No, currently there is no way. You could, however, restart just the neko binary inside room. So browser, X server and everything else stays running and all users should just reconnect.

Thank you, I'll try that. I was wondering if the banned IP addresses were stored in a file that Neko reads when someone tries logging in so that clearing bans would be a matter of editing/deleting that file, but I guess you're saying they're stored in a runtime variable. Is clearing bans a future option for the next version of Neko? Thanks again.

**Comment by m1k1o** [comment]
Yes, in the future neko will support accounts and they can be separately allowed to login or no.

**Comment by SpiderSuave** [comment]
> Yes, in the future neko will support accounts and they can be separately allowed to login or no.

Thank you. Are you referring to allowing to log in or not via the IP address attached to the registered user?

I hope accounts is just an added option and that everyone will still be allowed to login as guests like now with no sign-up required.

**Comment by m1k1o** [comment]
Yes, all of the current functionality will stay.
